ovirtlago/cmd.py

```diff
@@ -240,6 +241,19 @@ def do_ovirt_start(prefix, with_vms, **kwargs):
             with LogTask('Starting Engine VMs'):
                 prefix.virt_env.engine_vm().start_all_vms()
 
+    LOGGER.info(
+        '\n'
+        'oVirt environment is ready to be used.\n'
+        'You can access the web UI with the following link and credentials:\n'
+        'https://{ip}\n'
+        'Username: {username}\n'
+        'Password: {password}'.format(
```

Comment:
I think using """ would be nicer(and then you don't need the explicit eol'), i.e.:
```python
LOGGER.info("""

oVirt ....

""".format)
```